test/helpers/pkgMatchers.js

```diff
@@ -44,7 +90,40 @@ const matchDependency = dependencyType => {
   }
 };
 
+const X_OK = (fs.constants || fs).X_OK;
+
+const matchExecutableFile = () => {
+  return (pkgRef, raw) => {
+    const files = Array.isArray(raw) ? raw : [raw];
+    const expectation = `expected ${files.join(', ')} to be executable`;
+
+    const pkg = toPackage(pkgRef);
+
+    const failed = files.filter(file => {
+      try {
+        return fs.accessSync(path.join(pkg.location, file), X_OK);
+      } catch (_) {
+        return false;
```

Comment:
Shouldn't this be returning `true`, so it gets included in the filtered list? `fs.accessSync()` doesn't return anything, though in this case the default `undefined` is okay because it properly omits an executable file from the list of failed candidates.